docs/src/token.mdx

```diff
@@ -271,10 +854,14 @@ Updating 559u4Tdr9umKwft3yHMsnAxohhzkFnUBPAFtibwuZD9z
   New mint authority: disabled
 Signature: 5QpykLzZsceoKcVRRFow9QCdae4Dp2zQAcjebyEWoezPFg2Np73gHKWQicHG1mqRdXu3yiZbrft3Q8JmqNRNqhwU
 ```
+*Note*: Not currently available in the js/ts library
```

Comment:
This is actually possible! Strange, but possible. I added a test for it: https://github.com/solana-labs/solana-program-library/pull/2582

Reply:
Wow, I didn't see that instruction when I went looking for it. Updated with ts example